README.md

```diff
@@ -1,143 +1,405 @@
+
+<details>
+  <summary>See the fixtures version</summary>
+
+```yaml
+# test/fixtures/accounts.yml
+kaspers_donuts:
+  name: Kasper's Donuts
+
+# test/fixtures/users.yml
+kasper:
+  name: "Kasper"
+  email_address: "kasper@example.com"
+  accounts: kaspers_donuts
+
+coworker:
+  name: "Coworker"
+  email_address: "coworker@example.com"
+  accounts: kaspers_donuts
+
+# test/fixtures/menus.yml
+basic:
+  account: kaspers_donuts
+
+# test/fixtures/menu/items.yml
+plain_donut:
+  menu: basic
+  name: Plain
+  price_cents: 10_00
+
+sprinkled_donut:
+  menu: basic
+  name: Sprinkled
+  price_cents: 10_10
+```
+
+</details>
+
+### Oaken is like fixtures, we seed data before tests run
+
+The reason you go through all the trouble of massaging your fixture files is to have a stable named dataset across test runs that's relatively quick to load — so the database call cost is amortized across tests.
+
+Oaken mirrors this approach giving you stability in your dataset and the relative quickness to insert the data.
+
+For instance, if you have 10 tests that each need the same 2 records, Oaken puts them in the database once before tests run, same as fixtures.
+
+The tradeoff is that if you run just 1 test we'll still seed those 2 same records, but we'll also seed any other record you've added to the shared dataset that might not be needed in those tests.
+
+We rely on Rails' tests being wrapped in transactions so any changes are rolled back after the test case run.
+
+> [!NOTE]
+> It can be a good idea to structure your object graph so you won't need database records for your tests — reality can sometimes be far from that ideal state though. Oaken aims to make your present reality easier and something you can improve.
+
+### Oaken is unlike factories, focusing on shared datasets
+
+Factories can let you start an app easier. It's just this one factory for now, ok, easy enough.
+
+Over time, however, many teams find their factory based test suite slows to a crawl. Suddenly one factory ends up pulling in the rest of the app.
+
+Factories end up requiring a lot of diligence and passing just the right things in just-so to make managable.
+
+Oaken does away with this. See the sections on the fixtures comparisons above for how.
+
+> [!WARNING]
+> Full Disclaimer: while I have worked on systems using factories, I overall don't get it and the fixtures approach makes more sense to me despite the UX issues. I'm trying to support a partial factories approach here in Oaken, see the below section for that, and I'm open to ideas here.
+
+> [!TIP]
+> Oaken is compatible with FactoryBot and Fabricator, and they should be able to work together. I consider it a bug if there's compatibility issues, so please open an issue here if you find something.
```

Comment:
I like these! Noting that Oaken is compatible with FactoryBot and Fabricator helps encourage folks to adopt it